packages/apps/plugins/plugin-space/src/helpers.tsx

```diff
@@ -31,9 +36,34 @@ export class GraphNodeAdapter<T extends TypedObject> {
 
   constructor({ filter, adapter, propertySubscriptions, createGroup }: GraphNodeAdapterOptions<T>) {
     this._filter = filter;
-    this._adapter = adapter;
     this._propertySubscriptions = propertySubscriptions;
     this._createGroup = createGroup;
+
+    this._adapter = (parent, object, index) => {
+      const child = adapter(parent, object, index);
+
+      child.addAction({
+        id: 'delete',
+        label: ['delete object label', { ns: SPACE_PLUGIN }],
```

Comment:
nice!
in general i've been converting all of the labs plugins to just use "object" as the noun for all translations.